tests/unit/amo/components/TestSearchPage.js

```diff
@@ -23,4 +32,16 @@ describe(__filename, () => {
       });
     });
   });
+
+  it('should render Search results on search with query', () => {
+    const root = render({ filters: { query: 'burger' } });
+
+    expect(root.find(Search)).toHaveLength(1);
+  });
+
+  it('should render an error message on empty search', () => {
```

Comment:
There should be an extra test here for a search that's not `null` but is an empty string, eg: `const root = render({ filters: { query: '' } });`